packages/storybook/stories/sectional-banner.mdx

```diff
@@ -0,0 +1,89 @@
+import { SectionalBanner } from '@equisoft/design-elements-react';
+import { ArgTypes, Canvas, Meta } from '@storybook/blocks';
+import * as SectionalBannerStories  from './sectional-banner.stories';
+
+<Meta of={SectionalBannerStories} />
+
+# Sectional Banner
+1. [Definition](#definition)
+2. [Usage](#usage)
+3. [Variants](#variants)
+4. [Properties](#properties)
+
+## Definition
+A sectional banner is a notification element that displays a non-intrusive message inserted into a content area to
+communicate important information that may affect the user's experience.
+<Canvas of={SectionalBannerStories.Neutral} />
+
+## Usage
+### When to use
+- To communicate important information in a section of a screen.
+
+There are two major use cases for notifications: task-generated and system-generated [2] :
+
+- Task-generated banner: Task-generated banner are initiated in response to user action during a specific task. They
+  give users direct, immediate feedback. They should be placed in the region of the page the user is working in and
+  be related to the user’s action.
+    - e.g. To notify users of the status of an action or system [1].
+- System-generated banner : These banner are initiated by the application or system, independent of user action. They
+  provide updates on background system status or on events related to a specific section.
+    - e.g. To inform users of updates or changes to system status [1].
+
+### When not to use
+- When banners don't communicate important information that needs to be promoted [2].
+- When they are not directly related to the content or activity of the specific section [2].
+- When they are redundant. Avoid overloading sections with repetitive or redundant banners, which can lead to visual
+  fatigue and a degraded user experience [2].
+- When the message concerns the entire application and not a specific page. Use Global Banner instead.
+
+## Variants
+### Neutral
+Neutral banners are used to provide additional information to the user that doesn’t need to grab too much attention.
+This variation is used to inform the user about something that might help or impact their journey.
+- Can be dismissed by the user or the application, depending on the content.
```

Comment:
tu veux dire quoi par "dismissed by the application" ?

Reply:
Si c'est task generated pis le user fix les commentaires, la banner s'en va.